content/influxdb/v2.0/process-data/common-tasks/task-failure.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Notify on failed tasks
+seotitle: Notify on failed tasks.
+description: >
+  How to create a task to send a notification if a specific task fails.
+menu:
+  influxdb_2_0:
+    name: Notify on failed tasks
+    parent: Common tasks
+weight: 201
+v2.0/tags: [tasks]
+---
+
+Use a task to monitor other tasks and receive alerts on failure.
```

Comment:
...receive alerts when a task fails.

Reply:
We may want to clarify/verify whether this script creates an alert "check" only, and if it doesn't, we could add "next steps" and link to how to create an alert "email" or notification endpoints and rules.